### DISCUSSION TITLE
how can i apply for the  Octernships from Bangladesh?

### DISCUSSION THREAD

**Discussion by taiyebnirjhar** [community/community#50247] [GitHub Education] [answered]
is there any way to participate from Bangladesh

**Comment by samus-aran** [comment, marked as answer]
Hi @taiyebnirjhar  👋

Thank you for this request. We will gradually expand to more regions over time so do stay tune.

**Reply by taiyebnirjhar** [reply]
Thanks

**Comment by rjarman** [comment]
@samus-aran Why Octernships is not permitted in Bangladesh? If our neighboring country India is allowed to participate but why can't we?😕

**Reply by samus-aran** [reply]
Hi @rjarman 

We acknowledge that students are eager to participate in the Octernships across the world but you can imagine that this does take a lot of effort to put into place. Whilst we work hard to gradually expand to more regions over time we appreciate your patience until we can share more on any further expansion.

**Reply by rjarman** [reply]
Hello @samus-aran 
It's not about the expansion, I'm pointing that why Bangladesh is not selected for the initial eligible country criteria like others?🙁

**Comment by sakib412** [comment]
Is there any update about this?
